internal/pkg/aws/ec2/ec2.go

```diff
@@ -101,12 +128,19 @@ func (c *EC2) ListVPC() ([]string, error) {
 		}
 		vpcs = append(vpcs, response.Vpcs...)
 	}
-	var vpcNames []string
+	var VPCs []VPC
 	for _, vpc := range vpcs {
-		vpcNames = append(vpcNames, aws.StringValue(vpc.VpcId))
+		VPC := VPC{
+			ID: aws.StringValue(vpc.VpcId),
+		}
+		for _, tag := range vpc.Tags {
+			if aws.StringValue(tag.Key) == "Name" {
+				VPC.Name = aws.StringValue(tag.Value)
+			}
+		}
+		VPCs = append(VPCs, VPC)
```

Comment:
nit: this is really a preference-thing but I find gathering variables and then creating a struct to be the most readable-route as you can see all the fields that're being set. For example:
```suggestion
		var name string
		for _, tag := range vpc.Tags {
			if aws.StringValue(tag.Key) == "Name" {
				name = aws.StringValue(tag.Value)
			}
		}
		vpcs = append(vpcs, VPC {
			ID: aws.StringValue(vpc.VpcId),
			Name: name,
		})
```

Reply:
Ah, gotcha. Thanks. I'll make the change in ExtractVPC as well.